src/Templating/Helper/MediaHelper.php

```diff
@@ -23,6 +23,9 @@
  * MediaHelper manages action inclusions.
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * @deprecated Since version 3.10, will be removed in 4.0.
+ * NEXT_MAJOR : remove this class
```

Comment:
Should I just remove this class?

Reply:
Probably not in that PR, which should be kept minimal, although I think I've done that in the past…

Reply:
ok, will leave it for now then.

Reply:
Better on a cleanup pr indeed